template.html

```diff
@@ -0,0 +1,143 @@
+<div id='success' style='display:none;'>
+	<h2>The task has been successfully saved.</h2>
```

Comment:
Semantically h2 is used to indicate a title. This is not the case here : you just want to show, when some event occurs, a piece of text displayed like a block : a simple div is enough for this purpose.